addons/sale_project/views/project_task_views.xml

```diff
@@ -88,6 +97,9 @@
         <field name="model">project.task</field>
         <field name="inherit_id" ref="project.view_task_tree2"/>
         <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_id']" position="attributes">
+                <attribute name="context">{'res_partner_search_mode': 'customer'}</attribute>
+            </xpath>
```

Comment:
I'm also wondering this context is actually useful when `account` is installed but it is directly added in `industry_fsm` and some other apps that has nothing to do with `account`. so should we add this directly on any apps or apps that are `account` dependent?
IMO it should added when it is useful. may be @xavierbol has better opinion?

Reply:
Normally, we should just use a context key only if we are sure it can be used in that module. Here, it is indeed the case as @kcv-odoo explains. 
For instance, the `fsm_mode` context key should not be used in `project` module since the context will just be useful when the `industry_fsm` module will be installed and so normally the usage of that context has been removed in Project modules and just used in the FSM modules.

However, if we grep in the codebase to know if we use the context only the modules in which the account is in theirs dependencies then we will find the context is even used in the base. So, I don't really know. 🤔 

IMO, the best place is in that module but we can indeed discuss to take a decision for that. 🤔 🙂

Reply:
Hey @MissingNoShiny 
I moved context from `project` to `sale_project`(account dependant) app because at start that context used in `base` [Task](https://www.odoo.com/web#id=2049131&menu_id=4720&cids=2&action=4043&model=project.task&view_type=form) and it moved to account app recently 
Thanks :slightly_smiling_face:

Reply:
Hello @kcv-odoo ,

@damrOdoo will add a module called `project_account`, I think we could wait his task be merged to move this context in project_account module. 🙂

His task is in RD Review, so normally it will be merged this week. 😁